tools/create-genesis/Cargo.toml

```diff
@@ -0,0 +1,40 @@
+[package]
+name = "create-genesis"
+version = "0.1.0"
+authors = ["Cryptape Technologies <contact@cryptape.com>"]
+edition = "2018"
+
+[dependencies]
+serde = { version = "1.0", features = ["derive"] }
+serde_yaml = "0.8"
+json = "0.11.13"
+serde_json = "1.0.39"
+ethabi = "7.0.0"
+ethereum-types = "0.5.2"
+hex = "0.3"
+tiny-keccak = "1.4.2"
+libsecp256k1 = "0.2.2"
+clap = "2.33.0"
+
+hashable = { git = "https://github.com/cryptape/cita-common.git", branch = "develop" }
+libproto = { git = "https://github.com/cryptape/cita-common.git", branch = "develop" }
+proof = { git = "https://github.com/cryptape/cita-common.git", branch = "develop" }
+pubsub = { git = "https://github.com/cryptape/cita-common.git", branch = "develop" }
+
+core-executor = { path="../../cita-executor/core"}
+evm = { path="../../cita-executor/evm"}
+
+[features]
+default = ["secp256k1", "sha3hash", "rabbitmq"]
+secp256k1 = ["libproto/secp256k1", "proof/secp256k1"]
+ed25519 = ["libproto/ed25519", "proof/ed25519"]
+sm2 = ["libproto/sm2", "proof/sm2"]
+sha3hash = ["hashable/sha3hash", "libproto/sha3hash", "proof/sha3hash"]
+blake2bhash = ["hashable/blake2bhash", "libproto/blake2bhash", "proof/blake2bhash"]
+sm3hash = ["hashable/sm3hash", "libproto/sm3hash", "proof/sm3hash"]
+rabbitmq = ["pubsub/rabbitmq"]
```

Comment:
Why does this carte need mq, privatetx and evm-debug features?

Reply:
Not required, i will remove it later.

Reply:
and other feature, where are they used?

Reply:
```
default = ["secp256k1", "sha3hash"]
secp256k1 = ["libproto/secp256k1", "proof/secp256k1"]
ed25519 = ["libproto/ed25519", "proof/ed25519"]
sm2 = ["libproto/sm2", "proof/sm2"]
sha3hash = ["hashable/sha3hash", "libproto/sha3hash", "proof/sha3hash"]
blake2bhash = ["hashable/blake2bhash", "libproto/blake2bhash", "proof/blake2bhash"]
sm3hash = ["hashable/sm3hash", "libproto/sm3hash", "proof/sm3hash"]
```
These features are necessary. Example like this:

```
cita-executor/core/src/state/mod.rs

  #[test]
    fn ensure_cached() {
        let mut state = get_temp_state();
        let a = Address::zero();
        state.require(&a, false, false).unwrap();
        state.commit().unwrap();

        #[cfg(feature = "sha3hash")]
        let expected = "530acecc6ec873396bb3e90b6578161f9688ed7eeeb93d6fba5684895a93b78a";
        #[cfg(feature = "blake2bhash")]
        let expected = "da6a27e8063dd144a208f56f6b8dd6b3536ce6adbea93a1bcba95ce7fedb802c";
        #[cfg(feature = "sm3hash")]
        let expected = "63b984566f02930a2a959cef805ab0b17f68653a1430a2422e62f21a9f878cde";

        assert_eq!(state.root().lower_hex(), expected);
    }
```
The compilation of executor requires these features.

Reply:
This's needed by executor, how about create_genesis?

Reply:
Use `core-executor = { path="../../cita-executor/core", default-features = false}`

then delete all features.